utils/nctl/sh/scenarios/itst02.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+source "$NCTL"/sh/utils/infra.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# stalling consensus by stopping enough nodes. It
+# then restarts the nodes and checks for the chain
+# to progress.
+#
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: itst02"
+    log "------------------------------------------------------------"
+
+    # 0. Wait for network start up
+    do_await_genesis_era_to_complete
+    # 1. Allow chain to progress
+    do_await_era_change
+    # 2-4. Stop three nodes
```

Comment:
Maybe, before stopping nodes, we should verify that they're all running correctly and agree on the LFB.

Reply:
added a fuction to check across all of the nodes used in this test: https://github.com/CasperLabs/casper-node/pull/959/files#diff-8bd21e6750b7829a78940cd1d0405bfe8e4ba19e78acb53c085cb8cfba164d41R138